google/cloud/spanner/commit_options.h

```diff
@@ -98,6 +99,7 @@ class CommitOptions {
   // Note that CommitRequest.request_options.request_tag is ignored,
   // so we do not even provide a mechanism to specify one.
   bool return_stats_ = false;
+  absl::optional<std::chrono::milliseconds> max_commit_delay_;
```

Comment:
Or, you could just move this to the end.

Reply:
Done.